### PR TITLE
refact(docs): update README with community and badges links

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

**What this PR does**:
Update repo docs and add community links in project README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # OpenEBS CStor CSI Driver
```
Comment (kmova) [inline README.md:1]: n_: @prateekpandey14 -- let us change from CStor to "cStor" everywhere.

Comment (prateekpandey14) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -17,24 +19,22 @@ The current implementation supports the following for CStor Volumes:
 
 ### Prerequisites
 
-Before setting up OpenEBS CStor CSI driver make sure your Kubernetes Cluster
+Before setting up OpenEBS cStor CSI driver make sure your Kubernetes Cluster
```
Comment (kmova) [inline README.md:22]: @prateekpandey14 to maintain a single source for setup instructions. Can you remove the following sections: 

- Prerequisites
- Setup OpenEBS CStor CSI Driver
- Provision a cStor volume using OpenEBS cStor CSI driver
- Snapshot And Clone cStor Volume using OpenEBS cStor CSI Driver

You can instead mention that. 

```
Refer to https://github.com/openebs/cstor-operators for detailed usage instructions. 
```

Let this page/repo only focus on how does it work. 

Change the sections on `Snapshot And Clone cStor` to also document the steps in involved in how does it work.

Comment (prateekpandey14) [reply README.md]: Cleaned the mentioned section , will create an issue to add the section on `Snapshot and Clone cStor` workflow doc.

**Review by kmova — APPROVED** [review]

**Pull request merged**